### DISCUSSION TITLE
MLH LocalHackDay - Share mono-thread

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#1189] [Conference Notes]
<p align="center">
<img src="https://cdn.discordapp.com/icons/688852585071116289/a38c8577fc136812f41c027b63b4f53f.webp?size=128" alt="MLH Logo" />
<img src="https://uploads-ssl.webflow.com/5f2c6dba183e4e456dfef9ac/600fdb222e63d00fa67bebc4_2020-LHD-share-for-web-01.png" alt="LHD Share logo" width="128" />
<img src="https://user-images.githubusercontent.com/62864373/112763643-f0e8ab80-9022-11eb-8fb1-3e6fea000b45.png" alt="EddieHub text logo" width="555" />
</p>

## Description
This is a mono-thread for discussing anything regarding MLH LHD-Share.
This is an open space for introductions, notes, questions, queries, resource-linking, event-updates, Livestream-reminders and for sharing any cool projects made during the hackathon etc. (best of all this is open to all). Discussing here would also help in documenting our LHD-Share experience.

Have fun and Happy Hacking-! :)

<3
### #EddieHub
Discord- http://discord.eddiehub.org/
Github Org- https://github.com/EddieHubCommunity
Community Website- https://www.eddiehub.org/

### Eddie Jaoude(GitHub Star of the Year-2020)
Youtube Channel- https://www.youtube.com/c/eddiejaoude
Twitter- https://twitter.com/eddiejaoude
Linkedin- https://www.linkedin.com/in/eddiejaoude/
GIthub- https://github.com/eddiejaoude/

## Community Guidelines-
Kindly be vary of the Code Of Conduct and make this experience a positive and memorable one for everyone-!
**Our Code Of Conduct** - https://github.com/EddieHubCommunity/support/blob/aa13d4720de975620d8ea095f62e50b6eaff4b27/CODE_OF_CONDUCT.md
**Community Guidelines** -
https://docs.github.com/en/github/site-policy/github-community-guidelines

**Comment by Vyvy-vi** [comment]
Introduction Thread-
If you're new to the community or are just popping in here, dropping as a reply to this comment, would be awesome-!

**Reply by Vyvy-vi** [reply]
(here's an intro example)-
Hello, everyone..
I'm Vyvy-vi. I like hanging around the github org and discord chat and helping out the community.
I usually use Python, but I am trying to expand my knowledge and skills in multiple areas.
_Your friendly neighbourhood potato_ :)

My Github: https://github.com/Vyvy-vi
My Twitter: https://twitter.com/Vyvy_viM

<p align="center"><img target="_blank" src="https://img.shields.io/badge/-All_guilds_are_awesome-blue?style=for-the-badge&logo=github&logoColor=black"/></p>

**Reply by verma-kunal** [reply]
> Introduction Thread-
> If you're new to the community or are just popping in here, dropping as a reply to this comment, would be awesome-!

Hello, everyone..
I'm Kunal. This is my first-time in an MLH Hackathon. A little confused here 😅  but excited to learn as well !
Really excited to interact in the community and meet new people.

My Github: https://github.com/Zipher007
My Twitter: https://twitter.com/kverma_twt

**Reply by Nirbhayparmar** [reply]
I am also new to the MLH LHD share Hackathon. Can you just guide me through all this?

**Reply by starlightknown** [reply]
See below

**Reply by Divyaansh313** [reply]
Hello everyone,
I am Divyaansh Jain, 3rd year Computer Science student from India. First time in an MLH LHD. It's been great working with the community. Really excited to interact with everyone and contribute to the community.

My GitHub: [https://github.com/Divyaansh313](url)
My Twitter: [https://twitter.com/DivyaanshJain](url)

**Reply by mrunankpawar** [reply]
Hello folks, 
I'm Mrunank Pawar, a second-year undergrad student at SKNCOE Pune. I have been part of EddieHub from a previous MLH weeklong event which was LHD: Build. This inclusive guild welcomes everyone whether they are a newbie or seasoned player !!
**_Collaboration First, Code Second !!_**
Twitter: https://twitter.com/MrunankPawar
Instagram: https://www.instagram.com/mrunank_pawar/
LinkedIn: https://www.linkedin.com/in/mrunankpawar/

**Reply by RutvikJ77** [reply]
Hello everyone,
Rutvik here. I love the idea of **Collaboration 1st and code 2nd!** and have been hanging around in the discord lately trying to help the community and help everyone grow. I am usually a Pythonista but I like to screw around with other new technology as well :P
You can connect with me on:
**Twitter** - https://twitter.com/Rutvik_J27
**Linkedin** - https://linkedin.com/rutvikjoshi

**Comment by starlightknown** [comment]
LHD day 1
Folks you can get few easy points right here! 
How?
Here's how-
Go to [week-long challenges form](https://docs.google.com/forms/d/e/1FAIpQLScfIZg3JIYbp1QLOq2PJBT0N5rKGzoiSs4TPEiRyULFe4iyLQ/viewform) and complete the special check in code- LHDShareRyansCoffee
And post about LHD on socials and submit the links, follow MLH on socials, say hi in your guild and create your own gif challenge. Resource to create your own gif [giphy](https://giphy.com/create/gifmaker) on [day 1 form](https://docs.google.com/forms/d/e/1FAIpQLSc7yBHieaHKq1jyNieLrU1YPTaSfoUXbinbWuvU5ywbH8eoWw/viewform) join MLH's discord and done!

**Reply by imvisp** [reply]
cool :+1: , this is really helpfull to new members

**Reply by imvisp** [reply]
> LHD day 1
> Folks you can get few easy points right here!
> How?
> Here's how-
> Go to [week-long challenges form](https://docs.google.com/forms/d/e/1FAIpQLScfIZg3JIYbp1QLOq2PJBT0N5rKGzoiSs4TPEiRyULFe4iyLQ/viewform) and complete the special check in code- LHDShareRyansCoffee
> And post about LHD on socials and submit the links, follow MLH on socials, say hi in your guild and create your own gif challenge on [day 1 form](https://docs.google.com/forms/d/e/1FAIpQLSc7yBHieaHKq1jyNieLrU1YPTaSfoUXbinbWuvU5ywbH8eoWw/viewform) join MLH's discord and done!

whats for set up a project on github and light shere edition ?

**Comment by priyadarshu** [comment]
Actually I couldn't get the !Light last time.
If I'm right, we had to recreate a website with HTML and CSS. Unfortunately mine didn't work. I'm still wondering the reason 😓

**Reply by imvisp** [reply]
> a website with HTML and CSS. Unfortunately mine didn't work.

and whats for github project stuff ?

**Reply by priyadarshu** [reply]
We just need to create a repo in github.
Its just getting started with Github, if you check the event description here 
https://organize.mlh.io/participants/events/6732-set-up-a-project-on-github

**Reply by Vyvy-vi** [reply]
However, I would suggest maybe look into structuring a project(best-practices), setting up github actions, making labels or using git.
That way, you can benefit and learn the most from this experience.

**Reply by Vyvy-vi** [reply]
@priyadarshu Maybe there was some issue with their program in recognising if the content is HTML?
Can you share, how you went about with it?

**Comment by Nitesh-thapliyal** [comment]
Easy points that all can collect is by doing the following tasks:

- Join discord community,  take ss, and upload ss in weeklong form
- Post about MLH LHD in socials, Take ss and upload it in weeklong form( daily we can upload 2 social posts)
- Follow MLH in Social media, take ss and upload in Weeklong form
- Daily check-in
- Add Sessions check-in-code in the weeklong form( check-in-code is shown during the session )

**Comment by eddiejaoude** [comment]
Thank you 🤓 👍
